src/compojure/core.clj

```diff
@@ -102,6 +102,12 @@
       ((mw handler) request)
       (handler request))))
 
+(defn- add-route-information
```

Comment:
`assoc-route-info` would be a name more in keeping with the other functions in the namespace.